### DISCUSSION TITLE
Sprint 2023

### DISCUSSION THREAD

**Discussion by YooSunYoung** [scipp/scipp#3267] [Developers]
We already had two rounds of sprints in August(31-08-2023) and September(29-09-2023) with ``sciline`` and ``scitacean`` each.
They were both fun and useful (gave us better understanding of those projects) so we will continue running sprints of different projects regularly. (every 1 or 2 months)
Let's use this dicussion board as a thread to log our sprints.

**Comment by YooSunYoung** [comment]
# [``Sciline``](https://github.com/scipp/sciline) sprint
- Date: 31-08-2023
- Host: @SimonHeybrock 
- Participants: @YooSunYoung @jl-wynen @nvaytet 
- Picked up issues:
  - https://github.com/scipp/sciline/issues/27
  - https://github.com/scipp/sciline/issues/38
  - https://github.com/scipp/sciline/issues/48
- Retros:
  - All participants enjoyed the sprint and felt like having much better understanding of the ``sciline`` and host was happy with the contribution :)
  - We accidentally spent too much time on discussion rather than coding so next time we decided to avoid too deep discussion about a specific issue.
  - We would also like to do more pair-programming or group-programming next time.

**Comment by YooSunYoung** [comment]
# [``Scitacean``](https://github.com/SciCatProject/scitacean) sprint
- Date: 29-09-2023
- Host: @jl-wynen 
- Participants: @YooSunYoung @nvaytet @jokasimr 
- Picked up issues:
  - https://github.com/SciCatProject/scitacean/issues/87
  - https://github.com/SciCatProject/scitacean/pull/149/
  - https://github.com/SciCatProject/scitacean/issues/76
- Retros:
  - All participants enjoyed the sprint and felt like having much better understanding of the ``scitacean`` and host was happy with the contribution :) (Yes, I just copied the same line from ``sciline`` sprint retros...! But it is true!)
  - This time, we did more pair-programming and spent more time on coding, not just discussion like we promised last time.
  - We had to spend half of the time to learn about the project but we expect that it will be much easier next time, so it was valuable.

**Reply by jl-wynen** [reply]
All PR's that were opened during this sprint are labeled: https://github.com/SciCatProject/scitacean/pulls?q=label%3Asprint-scipp-2023-09+
